### PR TITLE
Reward functions

### PR DESCRIPTION
**Pull request by nicholaekim**
made the reward (penalize) functions

### CODE REVIEW

File: src/reward_functions.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import torch
+import torch.nn as nn
+
+
+class PenalizeModel(nn.Module):
```
Comment (IanPTan) [inline src/reward_functions.py:5]: avoid making it an nn module

**Review by IanPTan — COMMENTED** [review]
just make it a function no need to make it a class
